perf/yb-connect.yaml

```diff
@@ -4,10 +4,29 @@ services:
     container_name: kafka-connect
     ports:
       - 8083:8083
+      - 9875:9875
+      - 9876:9876
     environment:
       BOOTSTRAP_SERVERS: ${KAFKA_HOST}
       GROUP_ID: 1
       CONFIG_STORAGE_TOPIC: my-connect-configs
       OFFSET_STORAGE_TOPIC: my-connect-offsets 
       HOST_NAME: "0.0.0.0"
       ADVERTISED_HOST_NAME: ${DOCKER_IP}
+      KAFKA_JMX_PORT: '9875'
```

Comment:
Build the docker image by downloading jmx agent using curl. Follow the pattern here: https://github.com/debezium/debezium-examples/blob/main/monitoring/docker-compose.yml#L26

In the [Dockerfile](https://github.com/debezium/debezium-examples/blob/main/monitoring/debezium-jmx-exporter/Dockerfile), the base image is `debezium/connect`. Instead use `quay.io/yugabyte/debezium-connector:latest` as the base image.